manual/statements/built/README.md

```diff
@@ -1,5 +1,269 @@
 
-*Coming soon... In the meantime, see the javadoc for [QueryBuilder].*
+Built statements are generated via [QueryBuilder]'s Fluent API.  Use of Fluent API allows
+easier build of complex queries, as opposed to use of hardcoded query strings.
 
-[QueryBuilder]: http://docs.datastax.com/en/drivers/java/3.5/com/datastax/driver/core/querybuilder/QueryBuilder.html
\ No newline at end of file
+Note: The provided builders perform very little validation of the built query. There is
+no guarantee that a built query is valid, and it is definitively possible to create
+invalid queries.
+
+Queries built with `QueryBuilder` are executed the same way as other queries--via
+`execute` or `executeAsync`.  When query is built with inlined values, then it doesn't
+differ much from a statement specified as a string.  But it's also possible to build
+the query with bind markers inside it, and then convert it into a [prepared statement](../prepared/).
+
+### Basics
+
+Generation of `BuiltStatement` is easy--start by calling of one of the
+[QueryBuilder]'s methods of that represent the CQL's "verb": `select`, `update`, `delete`,
+`insertInto`, or `truncate`, provide required parameters, and then call "verb"-specific
+functions to form a complete CQL statement (like, `where`, `from`, etc.).  The statement's
+target table can be specified as a simple table name (if a default keyspace has been set
+when creating the `Session` object), as a combination of keyspace name and table name, or as
+a [TableMetadata] object.
+
+Note: The `QueryBuilder` doesn't provide support for the full set of CQL.  For
+DDL operations (`CREATE TABLE`, etc.) you can use the [SchemaBuilder].  To perform other
+operations you still need to use [simple statements](../simple/).
+
+### Selecting data
+
+Selection of data is quite simple--at minimum you need to provide a list of columns to
+select, and then specify  from which table to select these columns (you can also optionally
+specify a condition, as described in the next section):
+
+```java
+BuiltStatement selectAll1 = QueryBuilder.select("id", "t").from("test", "test");
+ResultSet rs = session.execute(selectAll1);
+for (Row row: rs) {
+   System.out.println(row);
+}
+```
+
+Note: The call `select("column1", "column2")` is really a shortcut for a chain of calls
+`select().column("column1").column("column2")`.
+
+Please note that you can't pass the `*` as column name to select all columns--if you do
+this, you'll get an exception about unknown column. To select all columns you either need to use
+`select` in combination with `all` function, or simply don't specify a list of columns:
+
+```java
+BuiltStatement selectAll2 = QueryBuilder.select().all().from("test", "test");
+BuiltStatement selectAll3 = QueryBuilder.select().from("test", "test");
+```
+
+Besides selection of the specific columns, there is also a possibility of calling of
+arbitrary CQL function by using the `fcall` (this is just example, don't do this on real
+data):
```

Comment:
I would say "calling arbitrary CQL function**s** by using the `fcall` **method**